src/community/rest-openapi/openapi/src/main/resources/org/geoserver/rest/openapi/1.0.0/config.yaml

```diff
@@ -280,6 +280,14 @@ components:
             hitsIgnoreMaxFeatures:
               type: boolean
               description: Flag that determines if WFS hit requests (counts) will ignore the maximum features limit for this server
+            getFeatureOutputTypeCheckingEnabled:
```

Comment:
Ah thanks! It's a community module, but it's also the one that has an actual schema for the response.

Reply:
Maybe you could also update the example at:
https://docs.geoserver.org/latest/en/api/#1.0.0/owsservices.yaml

Reply:
done